llvm/lib/Target/AIE/AIE2Schedule.td

```diff
@@ -98,10 +98,9 @@ def DC_WM_PORT : FuncUnit;
 
 
 // 11.4 Other Scalar Register Ports
-
-// TESTME-VLIW: Conflict between VEXTRACT and other instructions
-// in the mov slot
-def S_WM_PORT : FuncUnit;
```

Comment:
This feels like a "fix" commit. Maybe you could instead use the correct `RS_WM_PORT` port directly in the previous commits?

Reply:
Am not sure, should I merge `[[AIE2] Model write to S reg accurately](https://github.com/Xilinx/llvm-aie/pull/157/commits/1110c0dde3e82823d52e5e0c8b08546b8c53e7f2)` commit with `[[AIE2] Update instr. using II_MOV_SCL to use itinerary based on regClass](https://github.com/Xilinx/llvm-aie/pull/157/commits/8b60fc1a48e6f78f0c28d41c722bd2cbae41c0a2)`  ?

Reply:
Probably every commit where you touch `S_WM_PORT` or `R_WM_PORT` needs to be fixed